dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -896,6 +896,7 @@
           }
         },
         "confidenceScore": {
+          "x-ms-client-name": "confidence",
```

Comment:
is it regular to see x-ms-client-name? it seems very counterintuitive to for the client on a specific name.

Reply:
We need to remove this specific instance but, yes, it's not uncommon to see `x-ms-client-name` or an `x-ms-enum.name` that is different from the enum model name. This is based on SDK architects' feedback. Ideally, REST and the SDK APIs agree, but when it's too late or undesirable to change the REST API, this is the second best option so that all language generators benefit.

Basically, the order of preference for renames should be:

1. Swagger models or properties
2. x-ms-client-name or x-ms-enum.name in swagger
3. Swagger transforms in the specs repo's readme.md
4. Swagger transforms in the individual language repos' readme.md. These are often sharable, so one person could write them and everyone else can copy/paste. 3 is better, but there may be times - like in a crunch - this is easier/faster.
5. For .NET, we have a bunch of attributes we can use to rename classes or properties but I feel we should use this sparingly since it benefits no other languages and, most often, we'll want to be consistent (barring casing) across languages.

Reply:
Removed x-ms-client-name as field